### PR TITLE
Sparkle75

### PR DESCRIPTION
**Pull request by Duncid**
## Description

- Fixing DropdownMenu used in a custom way with the new Sparkle 75
- Transferring and updating Notification's visual style to Sparkle

## Risk

I would not test Notifications on my env (don't know how).
**It needs to be tested.**

### CODE REVIEW

File: front/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "initdb": "npx tsx admin/db.ts"
   },
   "dependencies": {
-    "@dust-tt/sparkle": "^0.2.73",
+    "@dust-tt/sparkle": "^0.2.75",
```
Comment (philipperolet) [inline front/package.json:16]: Updating both sparkle & front in the same PR cannot be done unfortunately, first we need to do sparkle, then front

File: front/components/sparkle/Notification.tsx [hunk 2]
```diff
@@ -99,41 +91,12 @@ export function Notification({ title, description, type }: NotificationType) {
       leaveFrom="translate-y-0 opacity-100"
       leaveTo="translate-y-16 opacity-0"
     >
-      <div className="pointer-events-auto flex items-start gap-3 rounded-md border border-structure-100 bg-structure-0 p-2 shadow-md">
```
Comment (philipperolet) [inline front/components/sparkle/Notification.tsx:102]: Looks like this if ok for you
image

image

**Review by philipperolet — CHANGES_REQUESTED** [review]
LGTM :pray: however we need 2 prs, sparkle first then front
Added visuals for notif, can show you other visuals if needed

File: front/package.json (hunk 1 above)
Comment (Duncid) [reply front/package.json:16]: @philipperolet it's not the case, Sparkle 75 has already been merged and deployed.

File: front/components/sparkle/Notification.tsx (hunk 2 above)
Comment (Duncid) [reply front/components/sparkle/Notification.tsx:102]: Yes, look's I know about :)
What i could not test is if they are properly triggered and dissmissable

File: front/package.json (hunk 1 above)
Comment (philipperolet) [reply front/package.json:16]: If it was, we wouldn't see the changes in the PR so there's something to dig here, let's check

File: front/components/sparkle/Notification.tsx (hunk 2 above)
Comment (philipperolet) [reply front/components/sparkle/Notification.tsx:102]: yep triggers & dismisses correctly

**Pull request merged**